base_custom_info/README.rst

```diff
@@ -26,7 +26,7 @@ expected for a given record.
 
 To define a template, you need to:
 
-* Go to *Settings > Custom Info > Templates*.
+* Go to *Custom Info > Templates*.
```

Comment:
I realized that if you have access to setting the templates and values, you should be able to manage them from their menu more comfortably, no need to be an admin for that. Given in v9 the trend is to have more dedicated menus, I'm just following it. Besides, this base module isn't related to any other by default.

I guess the increase of top level menus comes from the enterprise version, where they are beautiful icons in a smartphone-like home panel. After all, the problem is that the backend community interface is way old-fashioned. Nobody will care about them if some day OCA creates a good alternative :yum: